test/test_cuda.py

```diff
@@ -200,7 +200,9 @@ def tmp(t):
     ('mode', small_3d, lambda t: [],),
     ('mode', small_3d, lambda t: [1], 'dim'),
     ('remainder', small_3d, lambda t: [3], 'value'),
+    ('remainder', small_3d, lambda t: [-3], 'negative_value'),
     ('remainder', small_3d, lambda t: [small_3d_positive(t)], 'tensor'),
+    ('remainder', small_3d, lambda t: [0 - small_3d_positive(t)], 'negative_tensor'),
```

Comment:
oh i see. you are right, for LongTensor, neg doesn't seem to be defined for some reason. have to track that down.